Source/cmGlobalFastbuildGenerator.cxx

```diff
@@ -262,6 +262,25 @@ class cmGlobalFastbuildGenerator::Detail::FileContext
 class cmGlobalFastbuildGenerator::Detail::Detection
 {
 public:
+
+	static std::string GetLastFolderName(const std::string& string)
+	{
+		return string.substr(string.rfind('/'));
+	}
+
+	static bool HasParentExcludedFromAll( cmTarget* target )
```

Comment:
Sorry, one last change...

Lets make this "IsExcludedFromAll"
Then all good :)